docs: cleanup repo readme.md (#4205)
flybywiresim/a32nx: README.md

# <img src="https://raw.githubusercontent.com/flybywiresim/fbw-branding/master/svg/FBW-Logo.svg" placeholder="FlyByWire" width="400"/>

[![Discord](https://img.shields.io/discord/738864299392630914.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/UjzuHMU) 

> [![Twitter](https://img.shields.io/twitter/follow/FlyByWire?style=flat&logo=twitter)](https://twitter.com/FlybywireSim)

Reviewer: This is not our Twitter account!

Author: Oops, I will change it

> [![Twitter](https://img.shields.io/twitter/follow/FlybywireSim?style=flat&logo=twitter)](https://twitter.com/FlybywireSim)

Reviewer 2: Since you added the Twitter, you might as well add our YouTube/Facebook accounts

[![GitHub latest release version](https://img.shields.io/github/v/release/flybywiresim/a32nx.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
[![Github All Releases download count](https://img.shields.io/github/downloads/flybywiresim/a32nx/total.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
[![GitHub contributors](https://img.shields.io/github/contributors/flybywiresim/a32nx.svg?style=flat)](https://github.com/flybywiresim/a32nx/graphs/contributors)

## About

> The A32NX Project is a community-driven open source project to create a free Airbus A320neo in Microsoft Flight Simulator that is as close to reality as possible. Our modification aims to enhance the default A320neo by improving the systems depth and functionality to bring it up to payware-level, all for free.

Reviewer 3: The project aims to enhance the default A320neo. Our modification is not a thing that can aim for anything.

[![Instagram](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=instagram&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://instagram.com/flybywiresim)
[![YouTube](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=youtube&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.youtube.com/c/FlyByWire-Simulations)
[![Facebook](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=facebook&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.facebook.com/FlyByWireSimulations/)

[![GitHub latest release version](https://img.shields.io/github/v/release/flybywiresim/a32nx.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
[![Github All Releases download count](https://img.shields.io/github/downloads/flybywiresim/a32nx/total.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
[![GitHub contributors](https://img.shields.io/github/contributors/flybywiresim/a32nx.svg?style=flat)](https://github.com/flybywiresim/a32nx/graphs/contributors)

## About

> The A32NX Project is a community-driven open source project to create a free Airbus A320neo in Microsoft Flight Simulator that is as close to reality as possible. Our project aims to enhance the default A320neo by improving the systems depth and functionality to bring it up to payware-level, all for free.

Reviewer 4: Probably need to remove the mention of the default now.

The following aircraft configuration is currently simulated:

 ```
 Model:      A320-251N
 Engine:     CFM LEAP 1A-26
 FMGS:       Honeywell Pegasus II

>  FWC Std.:   H2F9C

Reviewer 4: both work, no need to change what's not broken

except maybe the CFM part

Pilot ID can be found in the Optional Entries section of the Dispatch Options page.

## Downloads

> **Warning**: The development (master) and experimental versions have been forked as a separate package/aircraft in MSFS, and will appear separate from the default A320neo. Existing Asobo A320neo liveries will be incompatible with the new package.

Reviewer: **issue**: This duplicates the line below it.
**issue**: Now all versions are separate from the default A320.

**Warning:** The development (master) and experimental versions have been forked as a separate package/aircraft in MSFS, and will appear separate from the default A320neo. Existing Asobo A320neo liveries will be incompatible with the new package.

### A32NX Installer

> Download the new A32NX installer where you can select either the Stable or Developer build, and install the mod directly into your community folder, [download here](https://api.flybywiresim.com/installer) ([source](https://github.com/flybywiresim/installer/)).

Reviewer 4: Development not developer

[Download the stable release here.](https://github.com/flybywiresim/a32nx/releases/latest/download/flybywiresim-a32nx.zip)

You can view the changelog on the [releases page.](https://github.com/flybywiresim/a32nx/releases)

#### Unstable Master Branch Build

This has the latest features, but is much more unstable, use at your own risk.

[Download development build here.](https://github.com/flybywiresim/a32nx/releases/download/vmaster/A32NX-master.zip)

[View information regarding the latest build here.](https://github.com/flybywiresim/a32nx/releases/tag/vmaster)

#### Unstable Master Branch Build (with custom autopilot)

> This version is the same as the regular master/development version. However, this version comes with the (work in progress) LNAV and flight plan manager. **Expect issues with navigation and flight planning.** No support will be provided via Discord.

Reviewer 5: Let's just avoid having to update the readme everytime experimental is used for something different.

```suggestion
#### Experimental Branch Build

This branch is used from time-to-time for experimental features under heavy development. It's content will vary over time. No support will be provided via Discord, although users of the branch are advised to keep themselves updated with development progress via Discord.
```

#### Unstable Master Branch Build (with custom autopilot)

> This version is the same as the regular master/development version, but with the W.I.P. custom fly-by-wire, autopilot and autothrust system. Expect issues with autopilot, engines, autothrust and the PFD if you intend to use this version. No support will be provided via Discord.

Reviewer 3: I think we could find a better adjective here. WIP is clunky, no matter whether you add the dots or not.

Reviewer 2: How about experimental?

Reviewer 6: Or just `work in progress`...?

See below for the location of your Community folder:

For the Microsoft Store edition AND/OR Gamepass edition:

> - Copy the "A32NX" folder into your community package folder. The folder is located in:

Reviewer 3: This change is completely unnecessary. If the change is intended to resolve ambiguity between the A32NX folder and the community package folder, it fails in doing so, because "the folder" is just as ambiguous as "it".

`C:\Users\[YOUR USERNAME]\AppData\Local\Packages\Microsoft.FlightSimulator_<RANDOMLETTERS>\LocalCache\Packages\Community`.

For the Steam edition:

> - Copy the "A32NX" folder into your community package folder. The folder is located in:

Reviewer 3: See above.

`C:\Users\[YOUR USERNAME]\AppData\Roaming\Microsoft Flight Simulator\Packages\Community`.

For the Boxed edition:

> - Copy the "A32NX" folder into your community package folder. The folder is located in:

Reviewer 3: See above.

### Please follow ALL steps in this README if you encounter any issues with installation before seeking support.

> Open the zip that you downloaded from one of the links above, and drag the```flybywire-aircraft-a320-neo```folder inside the zip into your Community folder.

Reviewer: ```suggestion
Open the zip that you downloaded from one of the links above, and drag the `flybywire-aircraft-a320-neo` folder inside the zip into your `Community` folder.

```

See below for the location of your Community folder:

For the Microsoft Store edition AND/OR Gamepass edition:
- Copy the "A32NX" folder into your community package folder. The path is:
`C:\Users\[YOUR USERNAME]\AppData\Local\Packages\Microsoft.FlightSimulator_<RANDOMLETTERS>\LocalCache\Packages\Community`.

For the Steam edition:
- Copy the "A32NX" folder into your community package folder. The path is:
`C:\Users\[YOUR USERNAME]\AppData\Roaming\Microsoft Flight Simulator\Packages\Community`.

For the Boxed edition:

> - Copy the "A32NX" folder into your community package folder. The path is:

Reviewer 4: `It is located in:` sounded better imo

Reviewer 2: Also, change the package name

Author: What do you mean? What is the correct package name?

Author: Got it, it's in the latest commit for this PR.

- Autopilot goes direct to RWY on APP (same with the default A320) [Workaround: Use DIR to a waypoint or selected heading]
- Upper ECAM displays wrong THR levers position / N1 rating.
- Newly added ASOBO A320 liveries are incompatible with the A32NX mod.
- Rudder keybindings not working (you have to set your keybinding to rudder axis right and left)
- Wing dips on landing (due to bad transition to direct law in flare, same with the default A320)  [Workaround use minimal aileron input on landing]
- Black screens/unable to start (conflict with another mod/livery or incorrect installation of the A32NX mod, use the [installer](https://api.flybywiresim.com/installer))

## License

The original contents of this repository are licensed under the MIT license.

**Microsoft Flight Simulator © Microsoft Corporation. The FlyByWire Simulations A32NX was created under Microsoft's "Game Content Usage Rules" using assets from Microsoft Flight Simulator, and it is not endorsed by or affiliated with Microsoft.**

## FAQ

**Q: Can I download the aircraft in the current state?**

A: Yes, see [downloads.](https://github.com/flybywiresim/a32nx#downloads)

**Q: Where is the plane in-sim?**

> A: The mod is now separate from the base A320neo aircraft and will show up as a new aircraft under the name "FlyByWire Simulations A320neo (LEAP)". If you want to uninstall it and revert to the original aircraft, just delete the A32NX folder in your community folder (paths for all editions are above).

Reviewer 4: I would just remove this section

Reviewer 7: Honestly, we should just move the FAQ to the docs site. Most end-users don't get that far down on GitHub, our front end solutions (website or docs site) would serve this better.

**Q: Can I download the aircraft in the current state?**

> A: Yes, please seek [Downloads](#Downloads).

Reviewer 3: This change doesn't make any sense. What does it mean to "seek Downloads"?

Author: Thanks for taking time out of your day to notice my mistakes that I made while correcting others. I will take a bit more time and fix them up to make sure they fit perfectly.

**Q: Can I download the aircraft in the current state?**

> A: Yes, see [downloads.](https://github.com/flybywiresim/a32nx#downloads)

Reviewer: ```suggestion
A: Yes, see [downloads.](#downloads)
```

**Q: How do we report bugs?**

A: You can report bugs to us via the [#help channel](https://discord.gg/snueqJjDUN) in the Discord server. Addtionally, you can start/create a GitHub issue (please make sure your issue doesn't already exist). 

**Q: Why is my version not the same as what I see others using?**

> A: We have two versions, the Stable and Developer (Master). The Stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The Developer build is updated daily and is a constant work in progress and although we test thoroughly each update, minor issues may occur from time to time. If you find this to be the case, you can report these issues in the [#help channel](https://discord.gg/snueqJjDUN) Discord or via GitHub Issues (check there is not an existing issue of the same nature as yours).

Reviewer 4: Channel is now called `#support`

Author: Roger, will change tomorrow.

**Q: How do we report bugs?**

> A: You can report bugs to us via the [#help channel](https://discord.gg/snueqJjDUN). Addtionally, you can start/create a GitHub issue (please make sure your issue is not already known and the same as another user). 

Reviewer 3: Why remove the Discord mention? How about "the #help channel on our Discord server".

Reviewer 2: I'd also change the wording 

```suggestion
A: You can report bugs to us via the [#help channel](https://discord.gg/snueqJjDUN). Addtionally, you can start/create a GitHub issue (please make sure your issue doesn't already exist). 

```

**Q: How do we report bugs?**

A: You can report bugs to us via the [#support channel](https://discord.gg/snueqJjDUN) in the Discord server. Addtionally, you can start/create a GitHub issue (please make sure your issue doesn't already exist). 

**Q: Why is my version not the same as what I see others using?**

> A: We have two versions, the Stable and Developer (Master). The Stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The Developer build is updated daily and is a constant work in progress and although we test thoroughly each update, minor issues may occur from time to time. If you find this to be the case, you can report these issues in the [#support channel](https://discord.gg/snueqJjDUN) Discord or via GitHub issues (check there is not an existing issue of the same nature as yours).

Reviewer 4: Same

> A: We have two versions, the Stable and Development (Master). The Stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The Developer build is updated daily and is a constant work in progress and although we test thoroughly each update, minor issues may occur from time to time. If you find this to be the case, you can report these issues in the [#support channel](https://discord.gg/snueqJjDUN) Discord or via GitHub issues (check there is not an existing issue of the same nature as yours).

Reviewer 8: Shouldn't we mention Experimental as well?

Author: I was thinking of adding it as well before making the last commit but I decided to just keep it off for now.
